src/scss/components/_SearchResultCard.scss

```diff
@@ -0,0 +1,7 @@
+mark {
+  @apply bg-alert-gold-light p-0.5 border border-alert-gold rounded;
+
+  .SearchResultCard .bg-dark-blue & {
+    @apply bg-gray-dark p-0.5 border border-gray-mid-dark rounded text-white;
+  }
+}
```

Comment:
That's a good point. I'm glad you brought that up; I had meant to comment on being uncertain of my organization. I originally had it in `typography.scss`, but then when adding the variant for the featured result, I thought it would be better to have them together and that it would rarely be used in another context, but I agree, I definitely wouldn't want someone to miss out because they didn't use our standard SearchResultCard. I'll split it up!

Reply:
@stephiescastle Updated and ready for final review; thanks!